supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesJobCachingActor.scala

```diff
@@ -0,0 +1,31 @@
+package cromwell.backend.impl.jes
+
+import akka.actor.{Props, ActorRef}
+import cromwell.backend.BackendJobCachingActor.BackendJobExecutionResponse
+import cromwell.backend.{BackendJobCachingActor, BackendJobDescriptor}
+import org.slf4j.LoggerFactory
+import scala.concurrent.Future
+
+object JesJobCachingActor {
```

Comment:
I tested an alternate name and its more specific but sounds like a mouthful:
JesJobCachedResultCopyingActor

If we define Caching as Cache Result Copying, then maybe its okay to use them interchangeably? Though I'm not sure if that's universally true for Cromwell.